src/responseCollection.js

```diff
@@ -15,12 +15,17 @@ export default function(request$) {
       },
       // Function returning Observable of item values.
       getItem(key) {
-        const initialValue = localStorage.getItem(key) || ``
+        const initialValue = localStorage.getItem(key)
 
-        return local$
+        const value$ = local$
           .filter((req) => req.key === key)
           .map((req) => req.value)
-          .startWith(initialValue)
+
+        if (initialValue) {
```

Comment:
Maybe we should check for initialValue===null explicitly so allow falsy values like `""` to be stored.